### ISSUE TITLE
Add "registration extension" and "authentication extension" to glossary

### ISSUE THREAD

**Issue by vijaybh** [w3c/webauthn#131]
See comment from =JeffH on #130

**Comment by equalsJeffH** [comment]
see: https://github.com/w3c/webauthn/pull/130#discussion-diff-67204868

wherein @vijaybh says:  Changed "signature extension" to "authentication extension" since we define registration and authentication to be our two ceremonies. I will open a new issue to add the terms to the glossary, since I am having trouble coming up with polished definitions right now.

**Comment by equalsJeffH** [comment]
hm.. In looking at the spec, I notice that although we have named our overall ceremonies "registration" and "authentication" (a ceremony includes the user as an explicit component of the model), the underlying API artifacts are named as variations on "credential" and "assertion" respectively, e.g.:
```
    Promise < ScopedCredentialInfo > makeCredential (     
        Account                                 accountInformation,
        sequence < ScopedCredentialParameters > cryptoParameters,
        BufferSource                            attestationChallenge,
        optional unsigned long                  credentialTimeoutSeconds,
        optional sequence < Credential >        blacklist,
        optional WebAuthnExtensions             credentialExtensions
    );

    Promise < WebAuthnAssertion > getAssertion (
        BufferSource                     assertionChallenge,
        optional unsigned long           assertionTimeoutSeconds,
        optional sequence < Credential > whitelist,
        optional WebAuthnExtensions      assertionExtensions
```
Thus I wonder about (re-)naming these two major extension subclasses as "credential extensions" and "assertion extensions".

**Comment by selfissued** [comment]
I believe that the current use of the tags [=registration extension=] and [=authentication extension=] accomplishes this goal.

**Comment by equalsJeffH** [comment]
well, not exactly :)    ...but overall "yeah".

what I was pointing out above is more in regards to the former  `credentialExtensions` and `assertionExtensions` params on makeCred() and getAssn(), which are now ensconsed as members in `ScopedCredentialOptions` and `AssertionOptions`.

So i do agree that this issue is addressed, but just not by our changes in PR #386.

closing.
